openstack/identity/federatedauth/opidconfig/requests.go

```diff
@@ -0,0 +1,86 @@
+package opidconfig
```

Comment:
suggest to rename the package and directory to "openid_connect".

btw, can we merge this package into **providers** package?